config/config2.xsd

```diff
@@ -149,11 +149,46 @@
     <xs:element name="extended-runtime-options">
         <xs:complexType>
             <xs:all>
+                <xs:element ref="ffmpegthumbnailer" minOccurs="0"/>
                 <xs:element ref="mark-played-items" minOccurs="0"/>
             </xs:all>
         </xs:complexType>
     </xs:element>
 
+    <xs:element name="ffmpegthumbnailer">
```

Comment:
When producing the fixtures to test, there was several missing XSD elements.  Cleanup and implement accurate XSD.